multiqc/modules/bcftools/stats.py

```diff
@@ -112,45 +128,56 @@ def parse_bcftools_stats(self):
                     'anchor': 'bcftools-stats',
                     'content': linegraph.plot(self.bcftools_stats_indels, pconfig)
                 })
+            # Make line graph of variants per depth
+            if len(depth_data) > 0:
+                import pprint
+                pprint.pprint(depth_data)
+                pconfig = {'id': 'bcftools_stats_depth', 'title': 'Variant depths',
+                           'ylab': 'Fraction of sites (%)', 'xlab': 'Variant depth',
+                           'ymin': 0, 'ymax': 100, 'categories': True}
```

Comment:
Ah ok, just had a play - if I try to make `bin_name` a float when collecting the data I get an error because of one final one called `>500`. Not sure that there's really a way to get around this nicely so best as you have it I guess (though plot isn't as pretty).